portfolio/src/main/webapp/index.html

```diff
@@ -14,5 +14,40 @@ <h1>My Portfolio</h1>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
     </div>
+    <!-- content div will be deleted or moved somewhere else -->
```

Comment:
nit: please add this as a TODO higher up, at the declaration of the content div.
Style for todos: https://google.github.io/styleguide/htmlcssguide.html#Action_Items